src/js/modal-dialog.js

```diff
@@ -153,55 +153,52 @@ class ModalDialog extends Component {
    *
    * @fires ModalDialog#beforemodalopen
    * @fires ModalDialog#modalopen
-   *
-   * @return {ModalDialog}
-   *         Returns itself; method can be chained.
    */
   open() {
-    if (!this.opened_) {
```

Comment:
wouldn't been better to not refactor this method and just remove the chaining capabilities for this PR.